### DISCUSSION TITLE
Duplicate dependants are displayed for estree-util-value-to-estree

### DISCUSSION THREAD

**Discussion by remcohaszing** [npm/feedback#671] [Website] [answered]
I found a bug on npmjs.com, but I couldn’t find an issue tracker for this.

On https://www.npmjs.com/package/estree-util-value-to-estree, click _Dependants_. The package `@stefanprobst/remark-extract-toc` is shown as a dependant 6 times.

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. I've opened a bug for this internally.
